### PR TITLE
Rails 5 AR workflow addition.

### PR DESCRIPTION
**Pull request by merqlove**
Ability to setup custom parent classes (i.e. `ApplicationRecord`). Tested with rails5 branch.

### CODE REVIEW

File: lib/acts_as_follower/follower_lib.rb [hunk 1]
```diff
@@ -3,12 +3,14 @@ module FollowerLib
 
     private
 
+    DEFAULT_PARENTS = [ActiveRecord::Base]
```
Comment (jcoyne) [inline lib/acts_as_follower/follower_lib.rb]: Can you set the DEFAULT_PARENTS to `ApplicationRecord` if it's Rails 5?

**Comment by jcoyne** [comment]
This is great work. Thank you for contributing.

File: lib/acts_as_follower/follower_lib.rb (hunk 1 above)
Comment (merqlove) [inline lib/acts_as_follower/follower_lib.rb]: Yep

**Comment by merqlove** [comment]
Changed logic back for `unless`.

**Pull request merged**